src/Ansatz/Chain.jl

```diff
@@ -406,6 +406,20 @@ function mixed_canonize!(::Open, tn::Chain, center::Site) # TODO: center could b
     return tn
 end
 
+"""
+    normalize_by_mixed_canonization(tn::Chain)
+
+Normalizes the input [`Chain`](@ref) tensor network by transforming it
+to a mixed-canonized form.
+"""
+function normalize_by_mixed_canonization(tn::Chain)
+    centersite = Site(((Quantum(tn) |> nsites) + 1) ÷ 2)
+    canonized = mixed_canonize(tn, centersite)
+    normalize!(select(Quantum(canonized), :tensor, centersite))
+
+    return canonized
+end
+
```

Comment:
My idea about this was that it is a choosable method inside `LinearAlgebra.normalize!`. Can you change it?
(It should perform inplace, so call `mixed_canonize!` instead).

Also, the root site (what you call `centersite`) should be an argument and can be any site in the MPS.